components/PopupMenu.js

```diff
@@ -14,7 +14,7 @@ const Popup = styled(Box)`
   border-radius: 8px;
   background: white;
   box-shadow: 0px 4px 8px rgba(20, 20, 20, 0.16);
-  z-index: 10;
+  z-index: 1000;
```

Comment:
I applied this because the `TimePeriod` popup was appearing under the fixed collective navbar. By default, it seems to me that all popups menus should be displayed above it.